lib/node_modules/@stdlib/stats/base/snanmeanpn/test/test.ndarray.js

```diff
@@ -106,15 +104,13 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		NaN   // 4
 	]);
 
-	N = floor( x.length / 2 );
-	v = snanmeanpn( N, x, 2, 0 );
+	v = snanmeanpn( 4, x, 2, 0 );
```

Comment:
```suggestion
	v = snanmeanpn( 5, x, 2, 0 );
```

The comments are the source of truth. You should revisit the tests to ensure that we are providing the correct number of values based on what the comments indicate. Here, that is 5 values.